src/dodal/devices/eiger.py

```diff
@@ -290,7 +306,7 @@ def do_arming_chain(self) -> Status:
                 self.set_odin_pvs,
                 self.set_mx_settings_pvs,
                 self.set_num_triggers_and_captures,
-                lambda: await_value(self.STALE_PARAMS_TIMEOUT, 0, 60),
+                lambda: await_value(self.stale_params, 0, 60),
```

Comment:
This typo looks like it should have broken the arming, not sure how it was working before if this was in there

Reply:
Yh, it's probably just not covered well in the tests